src/views/components/Title.ts

```diff
@@ -19,3 +19,30 @@ export const TitleH2 = styled.h1<{ fz?: string; mb?: string }>`
   color: ${({ theme }) => theme.colors.text};
   font-weight: 500;
 `;
+
+export const Header = styled.h1<{ fz?: string; mb?: string }>`
```

Comment:
If these titles are used only in modal it should not be here. here should be defined common titles for all pages